src/Tribe/Metabox.php

```diff
@@ -253,7 +262,19 @@ public function get_panels( $post, $ticket_id = null, ?string $ticket_type = nul
 		 */
 		do_action( 'tec_tickets_panels_after', $post, $ticket_id, $ticket_type );
 
-		return $panels;
+		/**
+		 * Filters the panels data.
+		 *
+		 * @since TBD
+		 *
+		 * @param array       $panels      The panels data.
+		 * @param int|WP_Post $post        The post object or ID context of the panel rendering.
+		 * @param int|null    $ticket_id   The ID of the ticket being rendered, `null` if a new ticket.
+		 * @param string      $ticket_type The ticket type being rendered, `default` if not specified.
+		 *
+		 * @return array The panels data.
+		 */
+		return apply_filters( 'tec_tickets_panels', $panels, $post, $ticket_id, $ticket_type );
```

Comment:
To be safe, we should type-case the return value.

```suggestion
		return (array) apply_filters( 'tec_tickets_panels', $panels, $post, $ticket_id, $ticket_type );
```